components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
```

Comment:
We no longer need to import `React`. 

```suggestion
import { useState } from "react";
```